Description: vk::createResultValue() does not exist, so skip the check

--- decaf-emu.old/src/libgpu/src/vulkan/vulkan_driver.h
+++ decaf-emu.new/src/libgpu/src/vulkan/vulkan_driver.h
@@ -28,12 +28,7 @@
 #define STRINGIFY(x) #x
 #define TOSTRING(x) STRINGIFY(x)
 
-// Evaluate f and if result is not a success throw proper vk exception.
-#define CHECK_VK_RESULT(x) do { \
-   vk::Result res = vk::Result(x); \
-   int tmp = 0; \
-   vk::createResultValue(res, tmp, __FILE__ ":" TOSTRING(__LINE__)); \
-} while (0)
+#define CHECK_VK_RESULT(x) (void)0
 
 namespace vulkan
 {
